docs/pages/update-spec/introduction.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Update Spec
+sidebar_title: EAS Update 0
+---
+
+version 0
+
+---
+
+## Introduction
+
+This is the specification for EAS Update a protocol for managing over the air updates to apps running on multiple platforms.
+
+### Conformance
+
+Conforming servers and clients must fulfill all normative requirements. Conformance requirements are described in this document via both descriptive assertions and key words with clearly defined meanings.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in the normative portions of this document are to be interpreted as described in [IETF RFC 2119](https://tools.ietf.org/html/rfc2119). These key words may appear in lowercase and still retain their meaning unless explicitly declared as non‐normative.
+
+A conforming implementation of may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+
+### Overview
+
+EAS Update is a protocol for managing over the air updates to an app running on multiple platforms. 
+In particular, it is a protocol for fetching an `update` where an `update` is defined as a [manifest](#manifest) together with the assets it references.
+
+An app running a conformant EAS Update client will load the most recent update saved by the client.
+
+In order to obtain an `update`:
+1. The client makes a [request](#request). 
+2. If the response body is a new manifest (checking the manifest ID is sufficient) it proceeds to download and store the assets referenced by the manifest.
+3. The client updates the local state according to any metadata provided by the response [headers](#headers)
+
+We anticipate the primary user of this spec will be companies who need to manage their own update server to satisfy internal requirements.
+
+## Request
```

Comment:
This describes a request for a manifest. We should also describe the request expected for an asset. (A GET request that contains a query parameter containing the `url` field of the asset in the manifest)